static/js/map.js

```diff
@@ -90,14 +92,20 @@ var onlyTriggerGyms
 var noExGyms
 var noParkInfo
 
-
 createjs.Sound.registerSound('static/sounds/ding.mp3', 'ding')
 
 
 var genderType = ['♂', '♀', '⚲']
 var unownForm = ['unset', 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', '!', '?']
 var cpMultiplier = [0.094, 0.16639787, 0.21573247, 0.25572005, 0.29024988, 0.3210876, 0.34921268, 0.37523559, 0.39956728, 0.42250001, 0.44310755, 0.46279839, 0.48168495, 0.49985844, 0.51739395, 0.53435433, 0.55079269, 0.56675452, 0.58227891, 0.59740001, 0.61215729, 0.62656713, 0.64065295, 0.65443563, 0.667934, 0.68116492, 0.69414365, 0.70688421, 0.71939909, 0.7317, 0.73776948, 0.74378943, 0.74976104, 0.75568551, 0.76156384, 0.76739717, 0.7731865, 0.77893275, 0.7846369, 0.79030001]
 
+var weatherArray = []
+var weatherPolys = []
+var weatherMarkers = []
+var weatherColors = ['grey', '#fdfd96', 'darkblue', 'grey', 'darkgrey', 'purple', 'white', 'black']
```

Comment:
Any reason we couldn't move this to the config for people to configure colors that they want? I've had the request from a couple different mappers just seems like a convenience.